docs/develop/typescript/index.mdx

```diff
@@ -14,9 +14,23 @@ tags:
 ---
 
 
+![TypeScript SDK Banner](/img/banner-typescript-temporal.png)
+
+:::info TYPESCRIPT SPECIFIC RESOURCES
 Build [Temporal Applications](/temporal#temporal-application) with the TypeScript SDK.
 
-This guide provides a comprehensive overview of the structures, primitives, and features used in [Temporal Application](/temporal#temporal-application) development.
+**Temporal TypeScript Technical Resources:**
+- [TypeScript API Documentation](https://typescript.temporal.io)
+- [TypeScript SDK Code Samples](https://github.com/temporalio/samples-typescript)
+- [TypeScript SDK GitHub](https://github.com/temporalio/sdk-typescript)
+
+**Get Connected with the Temporal TypeScript Community:**
+
+- [Temporal TypeScript Community Slack](https://temporalio.slack.com/archives/C01DKSMU94L)
+- [TypeScript SDK Forum](https://community.temporal.io/tag/typescript-sdk)
+- [Temporal In Action: Use Cases with Typescript](https://temporal.io/in-use?sdk=TypeScript#temporal-in-action)
+- [Temporal Meetups](https://www.meetup.com/pro/temporal-technologies/)
```

Comment:
The Temporal Meetups link only appears on the TypeScript page. Do we want it on all pages?